test/Microsoft.ML.AutoML.Tests/AutoFitTests.cs

```diff
@@ -60,5 +62,91 @@ public void AutoFitRegressionTest()
 
             Assert.True(result.RunDetails.Max(i => i.ValidationMetrics.RSquared > 0.9));
         }
+
+        [Fact]
+        public void AutoFitRecommendationTest()
+        {
+            // Specific column names of the considered data set
+            string labelColumnName = "Label";
+            string userColumnName = "User";
+            string itemColumnName = "Item";
+            string scoreColumnName = "Score";
+            MLContext mlContext = new MLContext();
+
+            // STEP 1: Load data
+            var reader = new TextLoader(mlContext, GetLoaderArgs(labelColumnName, userColumnName, itemColumnName));
+            var trainDataView = reader.Load(new MultiFileSource(GetDataPath(TestDatasets.trivialMatrixFactorization.trainFilename)));
+            var testDataView = reader.Load(new MultiFileSource(GetDataPath(TestDatasets.trivialMatrixFactorization.testFilename)));
+
+            // STEP 2: Run AutoML experiment
+            ExperimentResult<RegressionMetrics> experimentResult = mlContext.Auto()
+                .CreateRecommendationExperiment(5)
+                .Execute(trainDataView, testDataView,
+                    new ColumnInformation() { 
```

Comment:
Don't you need to set the Label column here? Or is it inferred because it is explicitly named `Label`?

Do we throw an exception of the Label column is not found?

Reply:
The default column name for label is "Label" and is implicitly specified.
We would throw if Label column is not found. That is why this test does not fail.

I will add the LabelColumnName specifically on the next commit.